internal/controller/controllers/clusterdeployments_controller.go

```diff
@@ -212,13 +212,7 @@ func (r *ClusterDeploymentsReconciler) Reconcile(origCtx context.Context, req ct
 			}
 			return r.updateStatus(ctx, log, clusterInstall, cluster, err)
 		} else {
-			// Delete Day1 Cluster
-			_, err = r.deregisterClusterIfNeeded(ctx, log, req.NamespacedName)
```

Comment:
Instead of disabling it lets depend it on `EnableDay2Cluster` 
We probably don't want to have two clusters with the same kube-key if you create a day2 cluster.

Reply:
Done

Reply:
done